### ISSUE TITLE
YAML Config Parsing Issues in Eggs (Indents being removed and : being replaced)

### ISSUE THREAD

**Issue by aleforge** [pterodactyl/panel#2693]
* Panel or Daemon: Panel
* Version of Panel/Daemon: 1.1.1 
* Server's OS: Ubuntu 18.04
* Your Computer's OS & Browser: Windows 10 Firefox

**Describe the bug**
When parsing a yaml file using the Configuration Files the parsing is changing `:` with `=`. It is also removing comments and indents in the yaml as well. For example for this yaml:

```
bedrock:
  address: 0.0.0.0
  port: 19132
```

When parsed with this configuration:

```
{
    {
        "config.yml": {
        "parser": "yaml",
        "find": {
            "address": "0.0.0.0",
            "port": "{{server.build.default.port}}"
        }
    }
}
```

Becomes this output:

```
bedrock=
address=0.0.0.0
port=25566
```

**To Reproduce**
1. Open an egg that parses yaml
2. Add a yaml modification to the Configuration Files
3. Start the server to parse the yaml
4. Open yaml to see the results

**Expected behavior**
The data to the right of the `: ` would be replaced without changing to `=` and removing indents in the yaml.

**Comment by DaneEveritt** [comment]
Can you confirm I got that example output edited correctly? Use three backticks not one to do a codeblock.

**Comment by aleforge** [comment]
Yes the example output is correct. I have attached the two yaml files as txt since github won't let me upload yaml.

[config_before_parsing.txt](https://github.com/pterodactyl/panel/files/5519533/config_before_parsing.txt)

[config_after_parsing.txt](https://github.com/pterodactyl/panel/files/5519535/config_after_parsing.txt)

Link to egg I am making that installs geyser in papermc. https://drive.google.com/file/d/1S4-Se7sGXTrKjlrJsdKsfwcf77-q1wAp/view?usp=sharing

**Comment by matthewpi** [comment]
Just to clarify your `Configuration Files` section for the egg is
```
{
    "config.yml": {
        "parser": "yaml",
        "find": {
            "address": "0.0.0.0",
            "port": "{{server.build.default.port}}"
        }
    }
}
```
?

**Comment by aleforge** [comment]
```
{
    "server.properties": {
        "parser": "properties",
        "find": {
            "server-ip": "0.0.0.0",
            "server-port": "{{server.build.default.port}}"
        }
    },
    "plugins/Geyser-Spigot/config.yml": {
        "parser": "yaml",
        "find": {
            "address": "0.0.0.0",
            "port": "{{server.build.default.port}}"
        }
    }
}
```
No it is the above as listed in the egg I provide in the link. It isn't replacing yaml correctly per the examples provided above.

**Comment by parkervcp** [comment]
Dual file editing/managing is broken last time I saw.

**Comment by aleforge** [comment]
Even running it with a single parser gets the same results where invalid yaml is output. So if you just run

```{
    "plugins/Geyser-Spigot/config.yml": {
        "parser": "yaml",
        "find": {
            "address": "0.0.0.0",
            "port": "{{server.build.default.port}}"
        }
    }
}
```

it still will miss format the yaml.
